### PR TITLE
Add migration script for objects

### PR DESCRIPTION
**Pull request by acbramley**
* Adds missing app label to secrets, services, routes, and pvcs.
* Adds all db creds to secrets.

To run:
```
go run ./migration/migration.go --namespace=PROJECT_NAMESPACE
```

You can also add a `--dry-run` flag to preview what it will do without making any changes to objects.

### CODE REVIEW

**Comment by acbramley** [comment]
Maybe this doesn't need to be in this repo...but it can stay in this branch until we run it on prod :)

**Comment by nicksantamaria** [comment]
If any config changes trigger a deploy, the script now waits for those to complete before progressing to the next environment.

File: migration/migration.go [hunk 1]
```diff
@@ -104,14 +105,36 @@ func main() {
...
+				break;
+			}
+
+			if dcRefresh.Status.ObservedGeneration == dcRefresh.Generation {
+				fmt.Println("Deployment completed!")
+				break;
+			}
+
+			fmt.Println("  ... waiting for deployment to complete ...")
+			time.Sleep(5 * time.Second)
```
Comment (pingers) [inline migration/migration.go:135]: Cool!

**Comment by nicksantamaria** [comment]
I'm happy with this one - going to merge it now.

**Pull request merged**